cases/admin.py

```
@@ -408,6 +410,8 @@ class Media:
         "description",
     ]
 
+    list_display_links = ["title"]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)